File: 1495 - Lista de Orden Creciente/problem_1495.cs
using System;

namespace problem_1495
{
    class Program
    {
        static void Main(string[] args)
        {
            int n = Convert.ToInt32(Console.ReadLine());
            int[] lista = new int[n];
            for(int i=0; i<lista.Length; i++)
            {
                lista[i] = Convert.ToInt32(Console.ReadLine());
            }
            Array.Sort(lista);
            foreach(int var in lista)
            {
                Console.WriteLine(var);
            }
        }
    }
}

File: 1495 - Lista de Orden Creciente/problem_1495.java
import java.util.Arrays;
import java.util.Scanner;


public class problem_1495 {
    public static void main(String[] args) {
        Scanner e = new Scanner(System.in);
        int n= e.nextInt();
        int[] lista = new int[n];
        for(int i=0; i<lista.length; i++)
        {
            lista[i] = e.nextInt();
        }
        Arrays.sort(lista);
        for(int var:lista)
        {
            System.out.println(var);
        }
    }
}
